ajax/libs/hyperform/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "hyperform",
+  "filename": "hyperform.min.js",
+  "description": "Capture form validation back from the browser",
+  "homepage": "https://hyperform.js.org/",
+  "author": {
+    "name": "Manuel Strehl",
+    "url": "http://www.manuel-strehl.de"
+  },
+  "keywords": [
+    "form",
+    "forms",
+    "input",
+    "html5",
+    "form validation",
+    "validation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperform/hyperform.git"
+  },
+  "license": "MIT",
+  "npmName": "hyperform",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": "**/*"
+    },
+    {
+      "basePath": "css",
+      "files": "**/*"
```

Comment:
@Boldewyn Under css folder, I'll suggest you to use `"hyperform*"` instead of `"**/*"`. Thanks.